### PR TITLE
feat: add slipstream cli install into deploy action

### PR DESCRIPTION
**Pull request by bw-bmbarga**
#### Install the slipstream cli before performing a sliptream deploy action

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -107,11 +109,38 @@ async function directoryExists(url) {
...
+async function installSlipstreamCLI() {
+  const commandExistsAlready = await commandExists('slipstream');
+
+  if (commandExistsAlready) {
+    return;
+  }
+
+  // install cli
+  core.startGroup('Installing the Slipstream CLI');
+  // for the future check that the binary exist
```
Comment (ahume) [inline lib/index.js:125]: Isn't this handled above now? Not sure what this comment means.

**Comment by ahume** [comment]
Now that the `installSlipstreamCLI` function exists, can this also be used in the `install-cli` action?

File: lib/index.js (hunk 1 above)
Comment (bw-bmbarga) [reply lib/index.js:125]: It is handled, I will remove it 👍

**Comment by bw-bmbarga** [comment]
> Now that the `installSlipstreamCLI` function exists, can this also be used in the `install-cli` action?

Yes, we can. I will do it

**Review by ahume — APPROVED** [review]

**Pull request merged**